auslib/db.py

```diff
@@ -405,7 +406,9 @@ def _insertStatement(self, **columns):
 
            @rtype: sqlalchemy.sql.express.Insert
         """
-        return self.t.insert(values=columns)
+        table_columns = {k: columns[k] for k in columns.keys() if k in self.table.c}
+        unconsumed_columns = {k: columns[k] for k in columns.keys() if k not in table_columns}
```

Comment:
Hm, do we have cases where we end up passing invalid columns for some queries? If we do, we should fix them (in a followup bug, maybe), and eventually raise an error for this.

Reply:
There are columns that targets to other tables. In the last version of SqlAlchemy, these columns results in a warning. After 0.8 the warning was turned to exception.
So I decided to  keep the suport to columns from other tables, to not touch in flows like this:
https://github.com/mozilla/balrog/blob/master/auslib/web/admin/views/scheduled_changes.py#L100
https://github.com/mozilla/balrog/blob/master/auslib/db.py#L1231

`base_what ` has conditions information.

Also, if I try remove these columns, notifications will fail, because `conditions` are returned when we select ScheduledChangesTable.